src/components/molecules/UiLoader/UiLoader.vue

```diff
@@ -118,36 +113,30 @@ const component = computed<LoaderComponent>(() => {
   };
   return components[props.type];
 });
-const outerComponent = computed<VNode>(() => (
-  props.transition
-    ? h(TransitionGroup, {
-      name: props.transition as string,
-    })
-    : h((_, { slots }) => (slots.default as Slot)())
+const transitionComponent = computed(() => (isIfTransitionType.value
+  ? UiLoaderNativeTransition
+  : UiLoaderTransition
 ));
+const transitionComponentAttrs = computed(() => {
+  const attrs = {
+    name: props.name,
```

Comment:
@ArkadiuszKrzymienski name will be added to transitionComponentAttrs and can be overridden by transitionAttrs